src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -69,7 +69,7 @@ public void resetData(ReadOnlyAddressBook newData) {
         requireNonNull(newData);
 
         setPersons(newData.getPersonList());
-        //activePerson = newData.getPersonList().get(0);
+        setAssignments(newData.getAssignmentsList());
```

Comment:
I think we should add ```activePerson = null``` after this line since no users' assignment list are displayed on the ui

Reply:
May I know which test case you failing? I made this change but all test case pass.

Reply:
21 test cases failed if include that line
![Untitled](https://user-images.githubusercontent.com/77260240/137612204-9d4921ab-4258-4e6c-9aa3-f5bfe0d56a2d.png)
![Untitled](https://user-images.githubusercontent.com/77260240/137612216-e2a730bd-78e3-4471-b679-a893008af7f9.png)